pettingzoo/mpe/_mpe_utils/simple_env.py

```diff
@@ -12,6 +12,7 @@
 from pettingzoo.utils.agent_selector import AgentSelector
 
 alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+DYNAMIC_RESCALING = True
```

Comment:
Could this not be a global variable and be an environment argument instead? Also default it to True so we don't break existing learned algorithms which utilize image space.

Reply:
Totally understandable, everyone deserves some time off! I have made it into an env argument and changed the default to False (I guess that's what you meant instead of True). 

Note: It required changes in a lot of places including documentation hence you'll see multiple files in the PR.

Reply:
Thanks for the help!